src/bokeh/io/notebook.py

```diff
@@ -13,6 +13,8 @@
 #-----------------------------------------------------------------------------
 from __future__ import annotations
 
+import os
+import urllib
```

Comment:
These two imports need to go under standard library imports section below.

Reply:
Got it.